decentraland/unity-explorer
# fix: updated avatar colors on session change

## What does this PR change?

Before when logging out and logging back in with a different account the color of eyes, skin and hair were not getting updated.
Now it should work correctly

...

## How to test the changes?



1. Launch the explorer
2. Login with one account
3. Sign out from the button in the menu
4. Sign in with a different account with different hair/eyes/skin colors
5. Verify it works correctly and the avatar in the scene has the correct colors

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
